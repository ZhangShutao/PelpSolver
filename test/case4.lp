not p :- K[0,1) p.
p :- K[0,1) not p.