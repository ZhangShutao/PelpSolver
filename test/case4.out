WorldView 1:
{[1.0]:{}}
WorldView 2:
{[1.0]:{p}}